dpgen/generator/run.py

```diff
@@ -799,17 +814,23 @@ def run_train(iter_index, jdata, mdata):
             os.path.join("old", "model.ckpt.data-00000-of-00001"),
         ]
     elif training_init_frozen_model is not None or training_finetune_model is not None:
-        forward_files.append(os.path.join("old", "init.pb"))
+        forward_files.append(os.path.join("old", "init%s" % suffix))
 
-    backward_files = ["frozen_model.pb", "lcurve.out", "train.log"]
-    backward_files += [
-        "model.ckpt.meta",
-        "model.ckpt.index",
-        "model.ckpt.data-00000-of-00001",
+    backward_files = [
+        "frozen_model%s" % suffix,
+        "lcurve.out",
+        "train.log",
         "checkpoint",
     ]
-    if jdata.get("dp_compress", False):
-        backward_files.append("frozen_model_compressed.pb")
+    if suffix == ".pb":
+        backward_files += [
+            "model.ckpt.meta",
+            "model.ckpt.index",
+            "model.ckpt.data-00000-of-00001",
+        ]
+        if jdata.get("dp_compress", False):
+            backward_files.append("frozen_model_compressed%s" % suffix)
```

Comment:
should handle the situation for pt.

For pt, the checkpoint file is `model.pt`; the compressed model file will be `frozen_model_compressed.pth`

Reply:
It should be `model.ckpt.pt`